README.md

```diff
@@ -399,6 +399,41 @@ export default  Ember.Route.extend({
 ```
 And they still take advantage of caching in the `shoebox`. No more redundant AJAX for already acquired data. Installation details are available in the addon [README](https://github.com/appchance/ember-cached-shoe#ember-cached-shoe).
 
+### Rehydration
+
+What is Rehydration?
+
+The rehydration feature means that the Glimmer VM can take a DOM tree
+created using Server Side Rendering (SSR) and use it as the starting
+point for the append pass.
+
+See details here:
+
+https://github.com/glimmerjs/glimmer-vm/commit/316805b9175e01698120b9566ec51c88d075026a
+
+In order to utilize rehydration in Ember.js applications we need to ensure that
```

Comment:
Don't you need to mention from which version it is available?

Reply:
Definitely!  I just added a small blurb about fastboot / ember.js requirements to address this concern.